common/src/main/java/bisq/common/crypto/PubKeyRing.java

```diff
@@ -42,7 +42,7 @@
 @Slf4j
 @EqualsAndHashCode
 @Getter
-public final class PubKeyRing implements NetworkPayload, UsedForTradeContractJson {
+public class PubKeyRing implements NetworkPayload, UsedForTradeContractJson {
```

Comment:
Might be used in p2p messages so must be final. I know its prob. not used now but further refactoring might use it... all NetworkPayload classes should be final.